src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-FSharp/Startup.fs

```diff
@@ -25,10 +27,14 @@ type Startup private () =
     member this.Configure(app: IApplicationBuilder, env: IHostingEnvironment) =
         if (env.IsDevelopment()) then
             app.UseDeveloperExceptionPage() |> ignore
+#if (!NoHttps)
```

Comment:
@javiercn Some templates do not have the RequiresHttps option.

Reply:
Add it. Make every template consistent.